src/nrniv/bgpdma.cpp

```diff
@@ -33,7 +33,6 @@ extern IvocVect* vector_arg(int);
 extern void vector_resize(IvocVect*, int);
 
 } // extern "C"
-extern void (*nrntimeout_call)();
```

Comment:
the comment in the file says:
> //not sure this is useful for debugging when stuck in a collective.

so I think this is not relevant anymore. (i.e. we do have other timeout mechanism in neuron, isn't it?)

Reply:
Done in the next commit of this PR